### DISCUSSION TITLE
No PDF upload!

### DISCUSSION THREAD

**Discussion by omar-dulaimi** [forem/forem#18360] [Feature Requests]
Hey,

I find it weird that there is no solution for uploading PDF documents like cheatsheets and other files.
I just spent over an hour trying to find a good solution. Embedding does not solve it for me, as I have to host it somewhere first.

But even so, the file won't show up as a preview, it will remain as a link.

So is there an undocumented way?

Uploading image by image will take a lot of time, since each image has to be uploaded manually one by one, no batch upload.

**Comment by zainjnhbgvcfdxs** [comment]
there is a serious issue with uploading PDF document files I also try but didn't upload them one by one image is uploaded but it takes a lot of time. how is uploading a [file](https://www.billoo.com)?
